cl/search/tests.py

```diff
@@ -1988,6 +2008,512 @@ def test_citation_sorting(self) -> None:
         self.assertEqual(cs, cs_sorted)
 
 
+def is_es_online(connection_alias="default"):
+    """Validate that ES instance is online
+    :param connection_alias: Name of connection to use
+    :return: True or False
+    """
+    with captured_stderr():
+        es = connections.get_connection(connection_alias)
+        return es.ping()
+
+
+@skipUnless(is_es_online(), "Elasticsearch is offline")
+class ElasticSearchTest(TestCase):
+    fixtures = ["test_court.json"]
```

Comment:
No fixtures, please. :)

Reply:
Yeah, actually this was not required anymore, removed.